### PR TITLE
Avoid use of 'cool' and 'guys' (closes #6107)

### LINKED ISSUE #6107: Replace informal/anachronistic language
Noticed use of 'cool' and 'guys' while replaying Northern Rebirth S5. While I recognise this campaign - and scenario in particular - is more light-hearted and/or fourth-wall-breaking in some of its dialogue, the following phrases didn't read well to me. (I also searched all campaigns for other instances of these words.)

### Proposed Changes
https://github.com/wesnoth/wesnoth/blob/4c9e016fa319eea34d2aa60e02f666a354625b89/data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg#L1169-L1170
I had Tallin find this signpost, so it just sounded weird to me (even accepting that he's an uneducated, albeit brave and inspiring, fighter). I propose '**Great**, let's loot some booty!' Or since `en_US` is default, I suppose it's fine to leave it as 'let's go loot'.

https://github.com/wesnoth/wesnoth/blob/4c9e016fa319eea34d2aa60e02f666a354625b89/data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg#L804-L805
This is in the context of Morvin's and Thera's first death-respawn event. Propose '...kept you **two** in jail...'

https://github.com/wesnoth/wesnoth/blob/4c9e016fa319eea34d2aa60e02f666a354625b89/data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg#L1840-L1841
Finding the signpost to Malifor's study. Propose 'This way, **everyone**!'

https://github.com/wesnoth/wesnoth/blob/4c9e016fa319eea34d2aa60e02f666a354625b89/data/campaigns/Northern_Rebirth/scenarios/10a_Stolen_Gold.cfg#L1054-L1055
The lich is getting impatient at the lengthy introductions between Tallin, Eryssa, and Hidel. Propose 'if you **lot** are quite finished...'

https://github.com/wesnoth/wesnoth/blob/4c9e016fa319eea34d2aa60e02f666a354625b89/data/campaigns/Son_Of_The_Black_Eye/scenarios/08_Silent_Forest.cfg#L271-L272
Even accepting the orcs' general lack of literary sophistication, 'guys' still sounds too anachronistic to me. Propose to just skip the word 'guys' altogether, as it still reads fine to me, ie 'You already have...'

https://github.com/wesnoth/wesnoth/blob/4c9e016fa319eea34d2aa60e02f666a354625b89/data/campaigns/Under_the_Burning_Suns/scenarios/03_Stirring_in_the_Night.cfg#L415-L416
Elyssa is the fire mage introduced in the prior scenario of Under the Burning Suns, and this is in the context of discussing the undead appearing during the 'long dark' night cycle. Propose 'why can't these **creeps** just stay dead?' (Elyssa also seems to be a less formal character, otherwise I'd consider 'ghosts' or similar.)

### Probably Justifiable
https://github.com/wesnoth/wesnoth/blob/4c9e016fa319eea34d2aa60e02f666a354625b89/data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg#L1037-L1038
Camerin's already a little crazy, so I suppose using 'cool' in his informal context is okay.

https://github.com/wesnoth/wesnoth/blob/4c9e016fa319eea34d2aa60e02f666a354625b89/data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg#L464-L465
https://github.com/wesnoth/wesnoth/blob/4c9e016fa319eea34d2aa60e02f666a354625b89/data/campaigns/Northern_Rebirth/scenarios/07a_Settling_Disputes.cfg#L832-L833
Four uses of 'guys' by Camerin in the one scenario. But I suppose if he's allowed to use 'cool' in his crazy informal dialogue, he can use 'guys' here, though I could probably rephrase this to avoid its use.

Every other use of 'cool' that I found relates to temperature rather than as a synonym for 'super', 'awesome', 'great', etc. Other uses of 'guys' are only internal (not player-visible from within the game itself), eg team: 'bad guys'.

@nemaara or anyone else, if this looks okay I'll submit a PR shortly. Or let me know of any suggestions of your own, for either the proposed changes or for Camerin, so they can be merged together.

### PR DESCRIPTION
**Pull request by Wedge009**
As discussed in #6107, have left 'guys' in Camerin's lines alone.

### CODE REVIEW

**Comment by CelticMinstrel** [comment]
I'm not sure if it's your fault or GitHub's, but the issue link in your commit message is broken… it seems to link to your fork instead of the main Wesnoth repo.

**Comment by nemaara** [comment]
Yea, reading them over, the NR ones especially feel too formal to me. Personally I wouldn't make this change, but depends on what your goal is.

**Comment by Wedge009** [comment]
I think it's GitHub. I haven't done anything differently but I've noticed it's been showing inconsistently recently to me.

**Comment by Wedge009** [comment]
@nemaara Can you highlight which ones you object to? Is it all of the Northern Rebirth ones?

File: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg [hunk 1]
```diff
@@ -802,7 +802,7 @@
 
         [message]
             speaker=Tallin
-            message= _ "Wow! That’s incredible. Now I understand why the bag of bones kept you guys in jail; he just simply couldn’t kill you!"
+            message= _ "Wow! That’s incredible. Now I understand why the bag of bones kept you two in jail; he just simply couldn’t kill you!"
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:805]: Just "you" would be fine.

File: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg [hunk 2]
```diff
@@ -1167,7 +1167,7 @@
 
         [message]
             speaker=unit
-            message= _ "The Treasury! Cool, let’s go loot some booty!"
+            message= _ "The Treasury! Excellent, let’s clear it out!"
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:1170]: This one reads realllllly formally to me.

File: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg [hunk 3]
```diff
@@ -1838,7 +1838,7 @@
 
         [message]
             speaker=unit
-            message= _ "There we go! This way, guys!"
+            message= _ "There we go! This way, everyone!"
```
Comment (nemaara) [inline data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:1841]: This one might be better as just "This way!"

File: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg (hunk 2 above)
Comment (Wedge009) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:1170]: I didn't think it was _that_ formal. Contrast 'The Treasury, what fortune! Let us raid the enemy's coffers, that we may add its wealth and resources unto our own!'

How about 'Excellent, let's clean out the gold!' Or do you prefer something closer to the original like 'Excellent, let's loot the gold' or even a direct copy substituting only 'Cool' for 'Excellent'? I was just considering @CelticMinstrel's opinion on this particular change.

Comment (nemaara) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:1170]: It's not really a big deal, I was just letting you know about it. My brain is off so I'm just going to approve and let you decide if you want to mess around with the wording or not.

**Review by nemaara — APPROVED** [review]

File: data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg (hunk 2 above)
Comment (Wedge009) [reply data/campaigns/Northern_Rebirth/scenarios/05a_01_The_Pursuit.cfg:1170]: All right, I'll go with 'let's clean out the gold' as something that's hopefully a little less formal-sounding for you.
